packages/votingV2/schema.graphql

```diff
@@ -1,42 +1,81 @@
+type Globals @entity {
+  id: ID!
+
+  "List of all users addresses"
+  userAddresses: [String!]!
+
+  "Cumulative stake from all stakeholders"
+  cumulativeStake: BigDecimal!
+
+  "Emission rate per second"
+  emissionRate: BigDecimal!
+
+  "Anual Voting Token emission at current emission rate"
+  anualVotingTokenEmission: BigDecimal!
+
+  "Higher index to process among all users"
+  maxNextIndexToProcess: BigInt!
+
+  countNoVotes: BigInt!
+
+  countWrongVotes: BigInt!
+
+  countCorrectVotes: BigInt!
+
+  users: [User!] @derivedFrom(field: "globals")
+}
+
 type User @entity {
   "Utility entity that links data from a single ethereum address. Id of the entity is the ethereum address itself"
   id: ID!
 
   address: Bytes!
 
+  "Earnings percentage from staking over a 12-month period, exclusive of slashing and compound interest"
+  anualPercentageReturn: BigDecimal!
+
+  "Earnings from staking over a 12-month period, exclusive of slashing and compound interest"
+  anualReturn: BigDecimal!
+
   "Number of price requests that this user has revealed a vote for, and therefore participated in as a voter"
   countReveals: BigInt
 
-  "Provides a lower-bound on # of votes that user has correctly voted for. User may not have retrieved rewards for all of their correct votes."
-  countRetrievals: BigInt
+  countNoVotes: BigInt!
+
+  countWrongVotes: BigInt!
 
-  voterActiveStake: BigDecimal
+  countCorrectVotes: BigInt!
 
-  voterPendingStake: BigDecimal
+  voterStake: BigDecimal
 
+  "Voter pending unstake during cooling-off period"
   voterPendingUnstake: BigDecimal
 
-  "Cumulative active stake from all stakeholders"
-  cumulativeActiveStake: BigDecimal
+  "Total rewards withdrawn in voting token"
+  withdrawnRewards: BigDecimal
 
-  "Cumulative pending stake from all stakeholders"
-  cumulativePendingStake: BigDecimal
+  "Total slashed applied to the user"
+  cumulativeSlash: BigDecimal
 
-  "Total rewards claimed in voting token"
-  claimedRewards: BigDecimal
+  "Total calculated slashed applied to the user"
```

Comment:
is "user" the right term? not "stakeholder"?

Reply:
I've kept the `User` entity so everything is described using this term

Do we want to replace `user` everywhere with `stakeholder` including the entity? 
In the contracts we use the term `voter` instead, maybe it's more consistent to use `voter` instead of `user`. What do you think @chrismaree @mrice32 ?